src/twisted/conch/scripts/ckeygen.py

```diff
@@ -30,7 +30,7 @@ def assignkeygenerator(keygenerator):
         supportedKeyTypes[keyType] = keygenerator
```

Comment:
Sorry, what I meant is that this line should be moved underneath the declaration of `wrapper`, and then should be changed to `supportedKeyTypes[keyType] = wrapper`.